gears/flywheel/presidio-analyzer.json

```diff
@@ -0,0 +1,51 @@
+{
+  "author": "Flywheel",
+  "cite": "https://microsoft.github.io/presidio/",
+  "command": "python run.py",
+  "config": {
+    "debug": {
+      "default": false,
+      "description": "Log debug messages",
+      "type": "boolean"
+    }
+  },
+  "custom": {
+    "flywheel": {
+      "suite": "PHI-Detection",
+      "show-job": true
+    },
+    "gear-builder": {
+      "category": "classifier",
+      "image": "flywheel/presidio-analyzer:0.1.0"
+    }
+  },
+  "description": "This gear utilizes Mircosoft's Presidio SDK which utilizes Natural Language Processing techiniques like Regex and NER to de-identify PII for subject and paitent data protection.",
```

Comment:
Can you make the following update:

* typo `paitent` -> `patient`
* explicit NER
* rephrase de-identify as "detect and report"

Reply:
Changes implemented